kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/filtered-resources/kjars-sources/webservice-project/src/main/resources/travelAgencyProcessWithHeaders.bpmn2

```diff
@@ -0,0 +1,265 @@
+System.out.println("[Flight Web Service Task] YOU will be flying soon..YIPEEE....." + resultFlight.getRatePerPerson()); 
+System.out.println("Leaving " + kcontext.getNodeInstance().getNodeName() +" Node");
+]]></drools:script>
+        </drools:onExit-script>
+      </bpmn2:extensionElements>
+      <bpmn2:incoming>_492148F7-AB4F-4064-A972-AAAAF2B0D5A0</bpmn2:incoming>
+      <bpmn2:outgoing>_E0F59797-6B00-4B4C-9E74-04FBCD671914</bpmn2:outgoing>
+      <bpmn2:ioSpecification id="_GgtfMYtEEeiO7cOOapYI8w">
+        <bpmn2:dataInput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_TaskNameInputX" drools:dtype="String" itemSubjectRef="__34658F5F-BF7B-41BB-99ED-AD655FED5A4F_TaskNameInputXItem" name="TaskName"/>
+        <bpmn2:dataInput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ParameterInputX" drools:dtype="com.jboss.soap.service.acmedemo.FlightRequest" itemSubjectRef="__34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ParameterInputXItem" name="Parameter"/>
+        <bpmn2:dataInput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_NamespaceInputX" drools:dtype="String" itemSubjectRef="__34658F5F-BF7B-41BB-99ED-AD655FED5A4F_NamespaceInputXItem" name="Namespace"/>
+        <bpmn2:dataInput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ModeInputX" drools:dtype="String" itemSubjectRef="__34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ModeInputXItem" name="Mode"/>
+        <bpmn2:dataInput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_UrlInputX" drools:dtype="String" itemSubjectRef="__34658F5F-BF7B-41BB-99ED-AD655FED5A4F_UrlInputXItem" name="Url"/>
+        <bpmn2:dataInput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_InterfaceInputX" drools:dtype="String" itemSubjectRef="__34658F5F-BF7B-41BB-99ED-AD655FED5A4F_InterfaceInputXItem" name="Interface"/>
+        <bpmn2:dataInput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_OperationInputX" drools:dtype="String" itemSubjectRef="__34658F5F-BF7B-41BB-99ED-AD655FED5A4F_OperationInputXItem" name="Operation"/>
+        <bpmn2:dataOutput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ResultOutputX" drools:dtype="com.jboss.soap.service.acmedemo.Flight" itemSubjectRef="__34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ResultOutputXItem" name="Result"/>
+        <bpmn2:inputSet id="_GgtfMotEEeiO7cOOapYI8w">
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ParameterInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_NamespaceInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ModeInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_UrlInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_InterfaceInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_OperationInputX</bpmn2:dataInputRefs>
+          <bpmn2:dataInputRefs>_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_TaskNameInputX</bpmn2:dataInputRefs>
+        </bpmn2:inputSet>
```

Comment:
I don't see the headers HEADER_prefix anywhere. where is it ?

Reply:
I messed up when saving the file, It is corrected now